src/System.Text.Json/src/Resources/Strings.resx

```diff
@@ -429,4 +429,16 @@
   <data name="EmptyStringToInitializeNumber" xml:space="preserve">
     <value>Expected a number, but instead got empty string.</value>
   </data>
-</root>
+  <data name="JsonObjectDuplicateKey" xml:space="preserve">
+    <value>Provided property name already exists.</value>
+  </data>
+  <data name="PropertyNotFound" xml:space="preserve">
+    <value>Property with name {0} not found.</value>
+  </data>
+  <data name="PropertyTypeMismatch" xml:space="preserve">
+    <value>Property with specified name has a different type than expected</value>
+  </data>
+  <data name="InvalidDuplicatePropertyNameHandling" xml:space="preserve">
+    <value>Provided manner of handling duplicates does not exist.</value>
```

Comment:
nit: consider using existing style of exception message (like JsonCommentHandling):
https://github.com/dotnet/corefx/blob/5050ee83755eb07494d7d4ff6d759bc056ceb371/src/System.Text.Json/src/Resources/Strings.resx#L246-L248

```suggestion
    <value>The DuplicatePropertyNameHandling enum must be set to one of the supported values.</value>
```